biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -818,6 +825,21 @@ subroutine UpdatePlantHydrLenVol(ccohort,site_hydr)
     real(r8) :: norm                         ! total root fraction used <1
     integer  :: nlevrhiz                     ! number of rhizosphere levels
 
+    ! added by Junyan May 29, 2020
+    real(r8) :: dbh                          ! the dbh of current cohort                                             [m]   
+    real(r8) :: dbh_0                        ! the dbh of the sappling at recuitment                                 [m]
+
+    real(r8) :: dbh_max                      ! the maximum dbh a PFT can have as observed                            [m]
```

Comment:
Do we always want to have a dbh_max? I'm curious about the consequences of imposing this for all possilbe allometries.

Reply:
Also, following up on this, I can't find an instance where dbh_max is used in this PR?

Reply:
As far as I know, this is the dbh when the roots reach their maximum rooting depth.  It looks like these local variables need their definitions cleaned up a little bit.